### ISSUE TITLE
Correct confusing status message in event filter

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#870]
This message is confusing:

```
events processed = 4402
events passed    = 4207
       rejection = 0.9557019536574285
Done running EvioToLcio!
```

The "rejection" here should actually be "acceptance" as it indicates the fraction of events which passed the filter rather than those which were rejected.

**Comment by cbravo135** [comment]
This code was originally introduced by @JeremyMcCormick about 7 years ago:
https://github.com/JeffersonLab/hps-java/blame/6ac87fbf6599cb477c9efe4a81b4da9c4e53a3b3/recon/src/main/java/org/hps/recon/filtering/EventReconFilter.java#L29

**Comment by cbravo135** [comment]
It would also be reasonable to leave it as "rejection" and use 1 minus the current value.

**Comment by JeremyMcCormick** [comment]
Alright which is more useful?

I can change it to 1 minus - val if that is preferable.

**Comment by cbravo135** [comment]
Doesn't matter to me, what is going on is obvious either way. I can do 1 - val in my head to get the other one so doing one IMO is the same as the other. 

tldr; they are equally useful.
